doc/hooks/breakpoint.section.md

```diff
@@ -10,9 +10,7 @@ nativeBuildInputs = [ breakpointHook ];
 When a build failure happens there will be an instruction printed that shows how to attach with `cntr` to the build sandbox.
 
 ::: {.note}
-::: {.title}
```

Comment:
Possibly because they were not nested properly. I would expect the following to be [recognized by the DocBook Writer](https://github.com/jgm/pandoc/blob/d9a7d473d990450cd6892d878b72b1dc4b79a51c/src/Text/Pandoc/Writers/DocBook.hs#L214-L217):

```markdown
:::: {.note}
::: {.title}
Title
:::

Admonition body
::::
```

But I never liked the syntax, would be better served by a heading.